### PR TITLE
IE11 fixes

### LINKED ISSUE #1187: Sandboxes don't load in IE11
# 🐛 bug report

## Description of the problem

IE11 does not load any sandbox, user is greeted only with a blank page. Console error shows `SCRIPT1005: Expected '('`. Seems to be related to generator functions not being transpiled since this is the problem spot:

image

## How has this issue affected you? What are you trying to accomplish?

I was attempting to use codesandbox to test the IE11 compatibility of another library but ended up finding out that codesandbox isn't compatible either.

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | PROD-1539347246-8f1f711
| Browser | IE11
| Operating System | Windows 10

### PR DESCRIPTION
**Pull request by lbogdan**
Fixes #1187 (hopefully for good 🙂, as we now check if sandbox assets are valid ES5).

*description to come*

### CODE REVIEW

File: packages/app/scripts/build.js [hunk 1]
```diff
@@ -133,7 +133,6 @@ function build(previousSizeMap) {
     if (stats.hasErrors()) {
       console.error('Error creating a production build:');
       console.error(info.errors);
-      ls - a;
```
Comment (lbogdan) [inline packages/app/scripts/build.js]: Somehow I managed to introduce this with the build improvements. 🙂

**Comment by armujahid** [comment]
Surprised to see we are still supporting IE11 :)

**Comment by christianalfoni** [comment]
@lbogdan You still working on this? :) Nothing scares you man, IE11 bug fixing...

**Comment by christianalfoni** [comment]
With Overmind we do not really support IE11 as it is based on proxies. I will close it and relieve you of this pain @lbogdan. Please reopen if you like pain 😂

**Pull request closed without merging**